Lib/asyncio/selector_events.py

```diff
@@ -735,7 +742,25 @@ def resume_reading(self):
         if self._loop.get_debug():
             logger.debug("%r resumes reading", self)
 
-    def _read_ready(self):
+    def _read_ready_get_buffer(self):
+        if self._conn_lost:
+            return
+
+        try:
+            buf = self._protocol.get_buffer()
+            with memoryview(buf) as mem:
```

Comment:
You shouldn't a `memoryview` here, just pass `buf` to `recv_into`.

Reply:
Right